packages/roosterjs-editor-dom/lib/index.ts

```diff
@@ -50,6 +50,7 @@ export { default as setColor } from './utils/setColor';
 export { default as matchesSelector } from './utils/matchesSelector';
 export { default as createElement, KnownCreateElementData } from './utils/createElement';
 export { default as moveChildNodes } from './utils/moveChildNodes';
+export { saveTableInfo } from './utils/tableFormatInfo';
```

Comment:
still not prefer export it.

This should be handled all by VTable. You can always let VTable call saveTableInfo() from VTable.applyFormat()